staker-rewards/generated-rewards/generate-merkle-root.ts

```diff
@@ -0,0 +1,34 @@
+import { parseBalanceMap } from '../merkle-distributor/src/parse-balance-map'
```

Comment:
What do you think about renaming `staker-rewards/generated-rewards` to `staker-rewards/distributor`?

When I saw `generated-rewards`, my first though was that it's the directory where we'll publish interval results.

Reply:
Sure, done in 3a123ba